File: Website/DesktopModules/Admin/ModuleCreator/Templates/C#/Web Service API/templateRouteMapper.cs
#region Copyright

// 
// Copyright (c) _YEAR_
// by _OWNER_
// 

#endregion

#region Using Statements

using DotNetNuke.Web.Api;

#endregion

namespace _OWNER_._MODULE_
{
    public class _MODULE_RouteMapper : IServiceRouteMapper
    {
        public void RegisterRoutes(IMapRoute mapRouteManager)
        {
            mapRouteManager.MapHttpRoute("_OWNER_._MODULE_", "default", "{controller}/{action}", new[] {"_OWNER_._MODULE_"});
        }
    }
} 

File: Website/DesktopModules/Admin/ModuleCreator/Templates/VB/Web Service API/templateRouteMapper.vb
#Region "Copyright"

' 
' Copyright (c) [YEAR]
' by [OWNER]
' 

#End Region

#Region "Using Statements"

Imports DotNetNuke.Web.Api

#End Region

Namespace [OWNER].[MODULE]
    Public Class [MODULE]RouteMapper
        Implements IServiceRouteMapper
        Public Sub RegisterRoutes(mapRouteManager As IMapRoute)
            mapRouteManager.MapHttpRoute("[OWNER].[MODULE]", "default", "{controller}/{action}", New () {"[OWNER].[MODULE]"})
        End Sub
    End Class
End Namespace
